service/collector.go

```diff
@@ -269,7 +269,10 @@ func (col *Collector) setupConfigurationComponents(ctx context.Context) error {
 				return
 			default:
 				col.logger.Warn("Config WatchForUpdated exited", zap.Error(err))
-				col.reloadService(context.Background())
+				err := col.reloadService(context.Background())
```

Comment:
style nit: 

```
if err := col.reloadService(context.Background()); err != nil {
   col.asyncErrorChannel <- err
}
```

Makes the err more self contained.

Reply:
Done